transports/azure-service-bus/legacy/native-integration.md

```diff
@@ -21,9 +21,13 @@ This document provides guidance on how to integrate NServiceBus endpoints with n
 
 The following points need to be taken into account when integrating
 
- 1. Both the ASB SDK and the transport have some assumptions about the format of exchanged `BrokeredMessage` instances. Refer to [Brokered Message Creation](brokered-message-creation.md) to learn more about these assumptions and how to align the sending and receiving endpoints at the wire level.
- 1. The transport assumes a specific layout of ASB entities depending on the selected topology; any non-NServiceBus endpoint is expected to use the correct entities for each purpose. In general, the following rule applies: queues are for sending, and topics are for publishing. To learn more about the layouts of the built in topologies, refer to [Azure Service Bus Transport Topologies](/transports/azure-service-bus/legacy/topologies.md).
- 1. By default, the transport creates its own entities when they don't exist in the namespace yet. But non-NServiceBus endpoints may require manual creation of entities. Refer to the [Azure Service Bus documentation](https://docs.microsoft.com/en-us/azure/service-bus-messaging/) for more information on available ASB SDK's and tools to perform these tasks.
+1. Both the ASB SDK and the transport have some assumptions about the format of exchanged `BrokeredMessage` instances. Refer to [Brokered Message Creation](brokered-message-creation.md) to learn more about these assumptions and how to align the sending and receiving endpoints at the wire level.
+
+1. The transport assumes a specific layout of ASB entities depending on the selected topology; any non-NServiceBus endpoint is expected to use the correct entities for each purpose. In general, the following rule applies: queues are for sending, and topics are for publishing. To learn more about the layouts of the built in topologies, refer to [Azure Service Bus Transport Topologies](/transports/azure-service-bus/legacy/topologies.md).
```

Comment:
What is an entity? If that actually is a correct ASB term then I would leave it but otherwise it needs explanation.

Reply:
I didn't add this, perhaps @SeanFeldman can confirm?

Reply:
Anything created in the namespace is an entity (queue, topic, subscription, rule).
Yes, that's an official ASB term.